src/styles/block-card.css

```diff
@@ -0,0 +1,90 @@
+/* #region Card */
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+}
+
+.card__box-img {
+  display: flex;
+  padding: 0 3px;
+  margin: 0 0 40px;
+}
+
+.card__img {
+  width: 100%;
+  height: 134px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__code,
+.card__reviews,
+.card__text-price {
+  font-weight: 400;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-code-price);
+  margin: 0 0 16px;
```

Comment:
try to set only `margin-bottom` inside elements. if some elements have default margins, you can just reset it for tag, not need to do it for each class
<img width="590" alt="image" src="https://user-images.githubusercontent.com/102411130/218674688-a8c40c9f-acbe-49b4-a833-c42b0a5d480d.png">